aspnetcore/mvc/razor-pages/razor-pages-authorization-conventions.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Razor Pages authorization conventions in ASP.NET Core
+author: guardrex
+description: Learn how to control access to pages with conventions at startup that authorize users and allow anonymous users to access individual pages or folders of pages.
+keywords: ASP.NET Core,Razor Pages,authorization,AllowAnonymousToPage,AllowAnonymousToFolder,AuthorizePage,AuthorizeFolder
+ms.author: riande
+manager: wpickett
+ms.date: 10/24/2017
+ms.topic: article
+ms.assetid: f65ad22d-9472-478a-856c-c59c8681fa71
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: mvc/razor-pages/razor-pages-authorization-conventions
+---
+# Razor Pages authorization conventions in ASP.NET Core
+
+By [Luke Latham](https://github.com/guardrex)
+
+One way to control access in your Razor Pages app is to use conventions at startup to authorize users and allow anonymous users to access individual pages or folders of pages. The conventions described in this topic automatically apply [authorization filters](xref:mvc/controllers/filters#authorization-filters) to control access.
```

Comment:
Consider splitting this 1st sentence into 2. It's a bit long as it stands now.

Reply:
I agree ... stand-by for a commit.